public/manifest.json

```
@@ -0,0 +1,30 @@
+{
+    "short_name": "Orbits",
+    "name": "Orbits",
+    "start_url": ".",
+    "shortcuts": [
+      {
+        "name": "",
+        "short_name": "",
+        "description": "",
+        "url": ""
+      }
+    ],
+    "icons": [
+      {
+        "src": "favicon.png",
+        "sizes": "192x192",
+        "type": "image/png",
+        "purpose": "any maskable"
+      },
+      {
+        "src": "favicon.png",
+        "sizes": "512x512",
+        "type": "image/png",
+        "purpose": "any maskable"
+      }
+    ],
+    "display": "standalone",
```

Weren't we doing fullscreen? 
```suggestion
    "display": "fullscreen",
```

---

I don't let me just research a bit about display types

---

https://superpwa.com/doc/web-app-manifest-display-modes/

I found this article and I think standalone is the right one full screen one will be be trouble some

---

Okay thanks